### PR TITLE
Homepage updates

### PR DESCRIPTION
**Pull request by anjenkin**
JIRA TICKET - https://federal-spending-transparency.atlassian.net/browse/DA-5047

Please note I made a few other changes - 

- left aligned the resource section heading

- centered aligned DTS on tablet (according to the mocks)

- added the alt text.  The following text was provided by the content team - https://pmeonpremise.bah.com/sites/data-transparency/Shared%20Documents/02%20Data%20Lab/SEO%20and%20Alt%20Text/Data%20Lab%20ALT%20Text%20and%20Graph%20Descriptions_2020.01.02.docx?csf=1&e=DUKp6S

### CODE REVIEW

File: src/page-sections/home/feature-tile/feature-tile.jsx [hunk 1]
```diff
@@ -38,3 +39,12 @@ const FeatureTile = (props) => {
...
 export default FeatureTile;
+
+FeatureTile.propTypes = {
+  href: PropTypes.string.isRequired,
+  heading: PropTypes.string.isRequired,
+  body: PropTypes.string.isRequired,
+  mobileBody: PropTypes.string.isRequired,
+  imgSrc: PropTypes.string.isRequired,
+  imgAlt: PropTypes.string.isRequired
+}
```
Comment (brett-varney) [inline src/page-sections/home/feature-tile/feature-tile.jsx]: Again, thanks for making these clear

File: src/page-sections/home/more-analyses-tiles/more-analyses-tiles.jsx [hunk 2]
```diff
@@ -60,7 +61,7 @@ const MoreAnalysesTiles = () => {
                 </div>
 
                 <div className={moreAnalysesStyles.imageContainer}>
-                  <img src={item.imageSrc} className={moreAnalysesStyles.image} />
+                  <img src={item.imageSrc} className={moreAnalysesStyles.image} alt={item.alt} />
```
Comment (brett-varney) [inline src/page-sections/home/more-analyses-tiles/more-analyses-tiles.jsx]: If the alt is blank (as above), don't we need role=presentation?

File: src/styles/index.scss [hunk 3]
```diff
@@ -61,7 +61,7 @@ ul {
 @mixin homepageTitle {
   text-align: left;
   color: #2272CE;
-  font-size: 1em;
+  font-size: 1rem;
```
Comment (brett-varney) [inline src/styles/index.scss]: Good catch!

**Comment by anjenkin** [comment]
I don't see 'The new 2019 data is here!' banner
**[ajb] I just double checked with Beth and we agreed to remove this text as the data is no longer new.**

Homelessness text is still about contracts
**[ajb] Good catch.  Changing.**

Contract Explorer text doesn't match mock
**[ajb] The Contract Explorer text should match the homepage in prod**

More Analyses component images don't have alt text or role=presentation, so I'm sure will fail 508 compliance
**[ajb] Hmm... I double checked and we have alt text for these.  Adding now...**

I'm confused if 992-1199 is considered desktop or tablet. This maintains the tablet view up to 1199 (as before), but everything's much larger and spaced out than the mocks indicate.
**[ajb] Hmmm... double checking with Danielle**

Phone view has different text for YGTAF than other views & mock
**[ajb] We should have the different text on mobile.  The mobile text should be the same as prod.  I'll double check the mobile text is correct.**

Mock shows Contract Explorer image & text reversed in tablet view
**[ajb] Good catch**

All More Analyses images' right sides aren't cropped in tablet & phone (probably not important, just doesn't match current mock)
**[ajb] Hmm... ok.**

Tablet & phone views' footer headings aren't all centered (not clear if this should be in-scope here)
**[ajb] I believe this is out of scope.**

File: src/page-sections/home/more-analyses-tiles/more-analyses-tiles.jsx (hunk 2 above)
Comment (anjenkin) [reply src/page-sections/home/more-analyses-tiles/more-analyses-tiles.jsx]: Added the alt text

File: src/page-sections/home/feature-row/feature-row.module.scss [hunk 4]
```diff
@@ -64,15 +71,21 @@ $landing-title-color: #4a4a4a;
...
-    padding-top: $margin-15;
-    padding-left: $margin-15;
-    margin-right: -24px;
+    //padding-top: $margin-15;
+    //padding-left: $margin-15;
+    //margin-right: -24px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```
[the remark this answers is not in the document]
Comment (anjenkin) [reply src/page-sections/home/feature-row/feature-row.module.scss]: removed

File: src/page-sections/home/feature-row/feature-row.module.scss [hunk 5]
```diff
@@ -40,6 +40,13 @@ $landing-title-color: #4a4a4a;
 }
 
 .feature-row {
+  .content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
```
[the remark this answers is not in the document]
Comment (anjenkin) [reply src/page-sections/home/feature-row/feature-row.module.scss]: removed

**Comment by xjackk** [comment]
Just a **small** aesthetic thing but this border here doesn't align - It doesn't align on prod either but could be a nice to have a super super quick change. The `min-height` on that element can go from `100%` to `95.5%` and it aligns nicely. 

Other than that, everything looks super awesome ~ 

975spacing

Here is how it looks on `localhost` :

1a2a3a4a5a

**Comment by brett-varney** [comment]
Now I'm getting a series of React errors:

Warning: Failed prop type: The prop `href` [also for title, subtitle and imgSrc] is marked as required in `MoreAnalysesTiles`, but its value is `undefined`.

I don't know if these were happening before.

**Comment by anjenkin** [comment]
All updates made except the height of the c & u section.  I played around with it a little.  We can log a separate ticket to address it while fixing bugs.

**Pull request merged**